backend/src/test/kotlin/org/loculus/backend/service/CompressionServiceTest.kt

```diff
@@ -6,22 +6,27 @@ import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 import org.loculus.backend.api.Organism
 import org.loculus.backend.api.OriginalData
+import org.loculus.backend.config.BackendConfig
 import org.loculus.backend.controller.DEFAULT_ORGANISM
 import org.loculus.backend.service.submission.CompressionDictService
 import org.loculus.backend.service.submission.CompressionService
 import org.loculus.backend.service.submission.DictEntry
 
 class CompressionServiceTest {
     private val compressionDictServiceMock = mockk<CompressionDictService>()
+    private val backendConfigMock = mockk<BackendConfig>()
 
     private val compressor = CompressionService(
         compressionDictService = compressionDictServiceMock,
+        backendConfig = backendConfigMock,
     )
 
     @Test
     fun `Round trip compress and decompress sequences in original data`() {
         val organism = Organism(DEFAULT_ORGANISM)
 
+        every { backendConfigMock.zstdCompressionLevel } returns 10
```

Comment:
The configs for the tests are otherwise all in `resources/backend_config*.json`, wouldn't it make sense to add `zstdCompressionLevel=10` there instead of mocking it here?

Reply:
Yeah it's probably not necessary to mock at all since the config already has a default set. Will try deleting it.

Reply:
(By ommitting it in the config, it might be setting it explicitly to null but not sure)

Reply:
(I'm happy to add a commit if you like!)